src/server/index.ts

```diff
@@ -70,6 +71,11 @@ export class Server implements IServer {
         dbName: 'cse'
       })
     };
+    router.stack.forEach(endpoint => {
+      // The express router stack is of type any[]
+      // tslint:disable-next-line
```

Comment:
`//tslint:disable-next-line: no-any no-unsafe-any`
This should illustrate the point that the endpoint is of type `any`. We can't really do much about certain any types being returned from libraries, so the occasional line disabling is necessaary.

Reply:
This has been resolved in the latest changes.